.travis.yml

```diff
@@ -40,6 +40,8 @@ matrix:
     # We cannot have empty -A selector, so the one which always will be fulfilled
     - NOSE_SELECTION=
     - NOSE_SELECTION_OP=not
+    # To test https://github.com/datalad/datalad/pull/4342 fix
+    - LC_ALL=ru_RU.UTF-8
```

Comment:
Wouldn't you have to generate these locales?

Reply:
good question on either they are in effect.  Let me do one more run with a TEMP commit reverting the changes to code -- then things better fail

Reply:
perfect, thanks

Reply:
> Let me do one more run with a TEMP commit reverting the changes to code -- then things better fail

Looking green.

Reply:
:-( so indeed need to install/generate them I guess. will try now